docs/source/howto/include/scripts/performance_benchmark_base.py

```diff
@@ -10,8 +10,9 @@
 @click.command()
 @options.CODE(required=False, help='A code that can run the ``ArithmeticAddCalculation``, for example bash.')
 @click.option('-n', 'number', type=int, default=100, show_default=True, help='The number of processes to submit.')
+@click.option('--daemon/--without-daemon', default=True, is_flag=True, help='Submit to daemon or run synchronously.')
```

Comment:
Optionally change default
```suggestion
@click.option('--daemon/--without-daemon', default=False, is_flag=True, help='Submit to daemon or run synchronously.')
```